config/routes.rb

```diff
@@ -1,6 +1,12 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  root 'users#index'
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  resources :users, only: [:index, :show] do
+    resources :posts, only: [:index, :show]
+  end
```

Comment:
- [ ]  I think you repeated these routes below. Let's kindly keep our codebase DRY :+1:

Reply:
Thanks for pointing this out.